[2022-11-30T09:59:28.475+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T09:59:23.382418+00:00 [queued]>
[2022-11-30T09:59:28.483+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T09:59:23.382418+00:00 [queued]>
[2022-11-30T09:59:28.483+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T09:59:28.483+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-30T09:59:28.483+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T09:59:28.491+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonSensor): waiting_for_data> on 2022-11-30 09:59:23.382418+00:00
[2022-11-30T09:59:28.493+0000] {standard_task_runner.py:55} INFO - Started process 1991 to run task
[2022-11-30T09:59:28.496+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pratice_pg', 'waiting_for_data', 'manual__2022-11-30T09:59:23.382418+00:00', '--job-id', '713', '--raw', '--subdir', 'DAGS_FOLDER/practice_pg.py', '--cfg-path', '/tmp/tmp4ayzbgvs']
[2022-11-30T09:59:28.496+0000] {standard_task_runner.py:83} INFO - Job 713: Subtask waiting_for_data
[2022-11-30T09:59:28.531+0000] {task_command.py:376} INFO - Running <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T09:59:23.382418+00:00 [running]> on host 481f60569de1
[2022-11-30T09:59:28.570+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pratice_pg
AIRFLOW_CTX_TASK_ID=waiting_for_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-30T09:59:23.382418+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-30T09:59:23.382418+00:00
[2022-11-30T09:59:28.575+0000] {python.py:68} INFO - Poking callable: <function pg_check_data at 0x7fae1734cc20>
[2022-11-30T09:59:28.580+0000] {base.py:71} INFO - Using connection ID 'pgdb' for task execution.
[2022-11-30T09:59:28.676+0000] {base.py:71} INFO - Using connection ID 'pgdb' for task execution.
[2022-11-30T09:59:28.678+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-30T09:59:28.683+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=pratice_pg, task_id=waiting_for_data, execution_date=20221130T095923, start_date=20221130T095928, end_date=20221130T095928
[2022-11-30T09:59:28.710+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-30T09:59:28.730+0000] {taskinstance.py:2623} INFO - 1 downstream tasks scheduled from follow-on schedule check
